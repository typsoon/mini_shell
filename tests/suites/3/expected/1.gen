../../../common/simple_sh_gen