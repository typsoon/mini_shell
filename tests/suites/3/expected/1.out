----------------- test.e4.1.A
Makefile
manager.c
proto.h
rs.c
rs.h
service.c
----------------- test.e4.1.B
alloc.c
break.c
const.h
exec.c
forkexit.c
getset.c
glo.h
main.c
Makefile
misc.c
mproc.h
param.h
pm.h
proto.h
signal.c
table.c
time.c
timers.c
trace.c
type.h
utility.c
----------------- test.e4.1.C
fs
init
Makefile
pm
rs
alloc.c
break.c
const.h
exec.c
forkexit.c
getset.c
glo.h
main.c
Makefile
misc.c
mproc.h
param.h
pm.h
proto.h
signal.c
table.c
time.c
timers.c
trace.c
type.h
utility.c
----------------- test.e4.1.D
Makefile
manager.c
proto.h
rs.c
rs.h
service.c
alloc.c
break.c
const.h
exec.c
forkexit.c
getset.c
glo.h
main.c
Makefile
misc.c
mproc.h
param.h
pm.h
proto.h
signal.c
table.c
time.c
timers.c
trace.c
type.h
utility.c
----------------- test.e4.1.E
alloc.c
break.c
const.h
exec.c
forkexit.c
getset.c
glo.h
main.c
Makefile
Makefile
manager.c
misc.c
mproc.h
param.h
pm.h
proto.h
proto.h
rs.c
rs.h
service.c
signal.c
table.c
time.c
timers.c
trace.c
type.h
utility.c
